documentation/general/analyzer-redirecting.md

```diff
@@ -28,21 +28,35 @@ Targeting an SDK (and hence also loading analyzers) with newer major version in
 
 - Note that when `IAnalyzerAssemblyRedirector` is involved, Roslyn is free to not use shadow copy loading and instead load the DLLs directly.
 
+- It is possible to opt out of analyzer redirecting by setting environment variable `DOTNET_ANALYZER_REDIRECTING=0`.
```

Comment:
Feel like this needs a caveat of "we do not support such configurations" or similar.